app/src/main/res/layout/fragment_home.xml

```diff
@@ -111,7 +110,7 @@
         </com.google.android.material.card.MaterialCardView>
 
         <com.google.android.material.card.MaterialCardView
-            android:id="@+id/cardView_home_UNUSED4"
```

Comment:
From here down, replace the contents of this file with the one from develop to prevent reverting changes already merged into develop